src/docs/releases/1.9.0.md

```diff
@@ -152,3 +152,22 @@ public class AdminMenu : INavigationProvider
     }
 }
 ```
+
+### Admin Routes
```

Comment:
Please add this section to https://docs.orchardcore.net/en/latest/docs/reference/modules/Admin/ too.